draft-ietf-tls-ticketrequests.md

```diff
@@ -58,15 +58,34 @@ for future connection attempts.
 
 As per {{!RFC5077}}, and as described in {{RFC8446}}, TLS servers vend clients an arbitrary
 number of session tickets at their own discretion in NewSessionTicket messages. There are
-two limitations with this design. First, servers choose some (often hard-coded) number
-of tickets vended per connection. Second, clients do not have a way of expressing their
+two limitations with this design.
+
+First, servers choose some (often hard-coded) number of tickets vended per
+connection.  Some servers (e.g. OpenSSL) return a different default number of
+tickets for session resumption than for the initial full handshake that created
+the session.  No static choice, whether fixed, or resumption-dependent is ideal
+for all situations.
+
+Second, clients do not have a way of expressing their
 desired number of tickets, which can impact future connection establishment.
 For example, clients can open multiple TLS connections to the same server for HTTP,
 or race TLS connections across different network interfaces. The latter is especially
 useful in transport systems that implement Happy Eyeballs {{?RFC8305}}. Since clients control
 connection concurrency and resumption, a standard mechanism for requesting more than one
 ticket is desirable.
 
+Third, we should note that the various tickets in the client's possession
+ultimately derive from an initial full handshake.  Especially when the client
```

Comment:
Indeed they don't in theory, but in practice they would, because any time the client sees an initial handshake, it should dump all previous tickets and store only the new ones. :-)

Reply:
I think I'll have addressed this as well.